docs/gathered-data.md

```diff
@@ -13,1040 +13,1878 @@ archives are packaged as `.tar.gz` files in `/var/lib/insights-operator`.
 
 ## APIRequestCounts
 
-BuildGatherApiRequestCounts creates a gathering closure which collects API requests counts for the
-resources mentioned in the alert provided as a string parameter
+Creates a gathering closure which collects API requests counts for the
+resources mentioned in the alert provided as a string parameter.
```

Comment:
We might want to look at the documentation for conditional gathering next. While this text probably makes sense as a developer documentation for the conditional gatherer, it doesn't really tell users what is actually gathered (how to find out/what to do to change what is gathered).

Reply:
This is actually a really good point. I think that depending on the changes since we do have a small number of conditionals gathered I can update it with a better description of this PR.